daml/Marketplace/Custody/Service.daml

```diff
@@ -12,17 +12,25 @@ template Service
     operator : Party
     provider : Party
     customer : Party
-    account : Account
   where
     signatory operator, provider, customer
 
     key (operator, provider, customer) : (Party, Party, Party)
     maintainer key._1
 
     controller customer can
+
+      nonconsuming RequestOpenAccount : ContractId Custody.OpenAccountRequest
+        with
+          label : Text
+        do
+          assertMsg "Account label cannot be empty" $ label /= ""
+          create Custody.OpenAccountRequest with ..
+
       nonconsuming RequestDeposit : ContractId Custody.DepositRequest
         with
           asset : Asset
+          account : Account
```

Comment:
@dorrit-du-da let's inform KC about this change as it will affect asset bridge

Reply:
Yes, that's correct - now there's no "default account" with a custodian anymore. I suppose one would need to select one for asset bridge deposits, and could use the `AccountInfo` contracts for that.